rdtools/normalization.py

```diff
@@ -11,7 +11,7 @@ class ConvergenceError(Exception):
     '''Rescale optimization did not converge'''
     pass
 
-def normalize_with_expected_power(pv, expected_power, irradiance, pv_input='power'):
+def normalize_with_expected_power(pv, power_sim, irradiance, pv_input='power'):
```

Comment:
Not sure what to do about this function.  pv-terms defines terms for specific quantities, but `pv` can be either power or energy.  I personally prefer `expected_power` over `power_sim`, but again it's consistent with pv-terms.  Also I missed `irradiance`, should be `poa_global`.

Reply:
let's go with `power_expected`

Reply:
stick with `pv`